C/drcp.c

```diff
@@ -0,0 +1,263 @@
+/* Copyright (c) 2014, Oracle. All rights reserved. */
+
+/* drcp.c */
+
+/* Oracle OCI Database Resident Connection Pooling (DRCP) Example */
+/* Christopher Jones, 2014 */
```

Comment:
Should that not be updated to 2022 at least?

Reply:
Updated

Reply:
What about the second 2014?